src/script/modules/get-cof-value/index.ts

```diff
@@ -0,0 +1,66 @@
+import settings from '../../lib/settings';
+import { getCofStore } from '../../utils/snapchat';
+
+let oldGetClientCofValue: any = null;
+let oldGetClientCofValueAndLogExposure: any = null;
+
+function handleStoreEffect(storeState: any) {
+
+  if (oldGetClientCofValue == null) {
+    oldGetClientCofValue = storeState.getClientCofValue;
+    storeState.getClientCofValue = function (...args: any[]) {
+      const mobileEnabled = settings.getSetting('SNAP_AS_MOBILE');
```

Comment:
let's assume none of these settings are enabled, whats the point in having this being patched?

```
if (settings.getSetting('SNAP_AS_MOBILE') || settings.getSetting('ADS_ENABLED') || settings.getSetting('PRIVATE_STORIES') || settings.getSetting('ALLOW_SNAP_VIEWING')) {
  return;
}
```